views/components/ConfigurationCard/index.jsx

```diff
@@ -73,7 +74,7 @@ const ConfigurationCard = ({
           {ownsConfig ? (
             <>
               <IconButton
-                onClick={() => openNewWindow(routes.editConfiguration(_id))}
+                onClick={() => onEditConfig(_id)}
```

Comment:
we should pass the entire config object so that this can be flexible for future needs. What happens if we need another attribute from a config in the event handler in the future? By passing the full config the handler can use whatever it needs.

Reply:
Thanks for bringing this up. The current EditConfig page uses the route param to fetch the config with an api call. We can handle this when we are up to refactoring the EditConfigPage

Reply:
I don't follow. The change I'm suggesting is changing this to `onEditConfig(config)` and in the parent component changing the handler to `const onEditConfig = (config) => navigate(routes.editConfiguration(config._id))`

Reply:
😕  we already have the _id destructured
`const { _id, name, owner, readers, type } = config`

Reply:
in this component. The entire object could be useful to the parent component.

Reply:
The parent is in charge of rendering the list, I can see this being useful to the share component but we aren't there yet

Reply:
Right now it's too early to tell, but we have this flexibility to change it in the future